lib/rate-limiter.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _ = require('lodash');
+
+let rateTable = {};
```

Comment:
There'll be an `execute` function which will actually utilize the rate limiter - this function needs the table (so bringing it out is a foresight)